lib/workers/repository/errors-warnings.ts

```diff
@@ -69,14 +70,11 @@ export function getDepWarningsPR(
   warningText = emojify(
     `\n---\n\n### :warning: Dependency Lookup Warnings :warning:\n\n`
   );
-  warningText += `Please correct - or verify that you can safely ignore - these lookup failures before you merge this PR.\n\n`;
-  for (const w of warnings) {
-    warningText += `-   \`${w}\`\n`;
+  if (dependencyDashboard) {
```

Comment:
Can you change the  `warningText` strings, so the final result looks like this on the PR body?

> Warnings were logged while processing this repo. Please check the Dependency Dashboard for more information.

> Please check the logs for more information.

Reply:
@betterPeleg